.github/workflows/kind-e2e-tests.yaml

```diff
@@ -71,6 +71,7 @@ jobs:
 
           # restart controller
           kubectl -n pipelines-as-code delete pod -l app.kubernetes.io/name=controller
```

Comment:
update..
```
 kubectl patch configmap -n pipelines-as-code -p "{\"data\":{\"bitbucket-cloud-check-source-ip\": \"false\"}}" \
          --type merge pipelines-as-code
```

Reply:
TODO:
* [ ]  cleanup
* [ ] cli changes

Reply:
what are the cli changes that would be needed?